src/main/java/ch/securify/CompilationHelpers.java

```diff
@@ -83,18 +88,39 @@ private static String readFile(String path) throws IOException {
         return new String(encoded, Charset.defaultCharset());
     }
 
-    static JsonObject compileContracts(String filesol) throws IOException, InterruptedException {
+    static JsonObject compileContracts(String filesol) throws IOException, InterruptedException, RuntimeException {
         ProcessBuilder p = new ProcessBuilder("solc", "--combined-json", "abi,ast,bin-runtime,srcmap-runtime", filesol);
 
         File f = File.createTempFile("securify_compilation_", ".json");
         f.deleteOnExit();
 
         final Process process = p.redirectOutput(f).start();
 
+        Thread t = new Thread(){
```

Comment:
Look into `ProcessBuilder.redirectError` for an easier way to get this done. I think we should just have all error output on the default stderr visible, which should be possible with `p.redirectError(Redirect.INHERIT)`

Reply:
Yeah I haven't touched java in about 6months so this was just getting it done with what I know